fuel-block-producer/src/block_producer/tests.rs

```diff
@@ -52,8 +52,18 @@ async fn cant_produce_at_genesis_height() {
 
 #[tokio::test]
 async fn can_produce_initial_block() {
-    // simple happy path for producing the first block
-    let ctx = TestContext::default();
+    // setup dummy genesis block
+    let genesis_height = 0u32.into();
```

Comment:
should we make setting a default genesis block part of the TestContext init? Seems like this is going to turn into lots of extra boilerplate unrelated to the actual test logic if we have to include this every time.